images/miq-app-frontend/docker-assets/manageiq-http.conf

```diff
@@ -12,7 +12,7 @@ Options SymLinksIfOwnerMatch
 <VirtualHost *:80>
   DocumentRoot /var/www/miq/vmdb/public
   Include conf.d/manageiq-redirects-ui
-  Include conf.d/manageiq-redirects-ws
+  Include conf.d/manageiq-redirects-api
```

Comment:
In the other repo, I think there was an ordering issue that forced me to put api before ui, so that should be done here as well.

Reply:
I imagine it wouldn't matter. This is working now and you didn't change the ordering in the PR where you made the name change right?

Reply:
I ensured the ordering was api, then ui, then websocket because that's what it was.  I remember @bdunne saying order was important (though I don't know why).